.eslintignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+apps
```

Comment:
I think we should keep eslint for apps.

We can follow our projects that already used TypeScript: https://github.com/mybigday/llama.rn/blob/f35545b92241d8cc9ee2b22b5ca4c02b395db9bb/package.json#L127-L203

Then the package also needs to be rewritten in TypeScript since it uses Flow, and it may need a build script.

Reply:
Sorry for delayed reply. This was so the PR didn't get huge. I was going to do Typescript rewrite in another PR.